src/javascript/app/common/content_visibility.js

```diff
@@ -164,6 +166,7 @@ const ContentVisibility = (() => {
         if (Array.isArray(arr_mt5fin_shortcodes)) {
             if (arr_mt5fin_shortcodes.some(el => mt5fin_rules.includes(el))) show_element = !is_exclude;
         }
+        if (options_blocked && Client.isOptionsBlocked()) show_element = !is_exclude;
```

Comment:
Just a suggestion for more readability 
```suggestion
        if (options_blocked && Client.isOptionsBlocked()) { 
          show_element = !is_exclude;
        }
```

Reply:
Okay, done! 👍